source/index.md

```diff
@@ -372,18 +380,29 @@ Example response (snippet):
        "code": 20000
    },
    "processed_at": "2014-07-09T13:33:44+00:00",
-   "recurring": true
+   "recurring": true,
+   "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
 }
 ````
 
-Above can be repeated whenever you need to reserve money and must be followed
-by a capture transaction.
+This should (in case of non-zero amount) be followed by a capture.
+
+Subsequent authorizations are made similarly, but obviously cannot include
```

Comment:
Maybe `would not` works here as well, instead of `cannot`. In the sense that, subsequent _would not_ have the cardholder present.

Reply:
> Subsequent authorizations are made similarly, but obviously neither CSC nor PARes can/would be included. Use the [cards](#cards) endpoint or, when approved for CSC-less transactions, the `card` payment method.